service/mcs/mcs.go

```diff
@@ -37,6 +38,7 @@ type Namespace struct {
 	Namespace          *string           `json:"namespace,omitempty"`
 	Cost               *float64          `json:"cost,omitempty"`
 	ComputeCost        *float64          `json:"computeCost,omitempty"`
+	EbsCost            *float64          `json:"ebsCost,omitempty"`
```

Comment:
```suggestion
	EBSCost            *float64          `json:"ebsCost,omitempty"`
```

[See](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).